CanDrive.js

```diff
@@ -0,0 +1,7 @@
+var FirstName = "Aissatou";
+var LastName = "Diallo";
+var Age = 23;
+var drivingAge = 16;
```

Comment:
Don't use var. Use either const or let

Also a space after groups of variable declarations is good.